### DISCUSSION TITLE
Row level security

### DISCUSSION THREAD

**Discussion by marc-on-github** [planetscale/discussion#157] [Q&A] [answered]
We're currently considering migrating an application from PostgreSQL. What is the best way to implement row level security? I am grateful for any tips.

**Comment by jgreet** [comment, marked as answer]
Hi @Marc-Hoch, I'm afraid there isn't an equivalent row level security feature in Vitess and MySQL. Perhaps that's something we could look into in the future. 

If you're relying on RLS for authorization you'll have to move that logic to your application.
